src/models/SourceTag/index.ts

```diff
@@ -10,11 +10,13 @@ export interface SourceTag {
 
 /**
  * An enumerator which {@link SourceTags} uses to define the color of the tag rendered on the website.
- * Info is blue, success is green, warning is yellow and danger is red.
+ * Five types are available: blue, green, grey, yellow and red, the default one is blue.
+ * Common colors are red for (Broken), yellow for (+18), grey for (Country-Proof)
  */
 export enum TagType {
-    WARNING = 'warning',
-    INFO = 'info',
-    SUCCESS = 'success',
-    DANGER = 'danger'
+    BLUE = 'blue',
```

Comment:
What text do you need on the website to render the colors? I assumed that 'Danger' was red, and so on. Instead of changing to 
`GREEN = green` should we enum like this instead?

```BLUE = 'info',
RED = 'danger'```

You'd know how the rendering works better than I. If what you've listed is convenient for you, it sounds good to me.

Reply:
To render the color I indeed need the `danger`, `success` strings.
I just had one concern, this would mean in the versioning file the text would still be `danger` or `success`. If you believe it's not a real issue as users should not read the versioning file themself, your suggestion would make the rendering even easier as I would just pass the labels directly.

Reply:
As long as things are enforced in the enumeration, the versioning file contents don't matter. As long as they can be understood by the website, and possibly the app in the future.

Reply:
Then, it's perfect! Thanks, things will be even easier on the website. 
The BLUE tag is an empty string as it's the default Element tag type.